Doc/using/configure.rst

```diff
@@ -746,8 +746,8 @@ Example on Linux x86-64::
 At the beginning of the files, C extensions are built as built-in modules.
 Extensions defined after the ``*shared*`` marker are built as dynamic libraries.
 
-The :c:macro:`PyAPI_FUNC()`, :c:macro:`PyAPI_API()` and
-:c:macro:`PyMODINIT_FUNC()` macros of :file:`Include/pyport.h` are defined
+The :c:macro:`PyAPI_FUNC()`, :c:macro:`PyAPI_DATA()` and
```

Comment:
```suggestion
The :c:macro:`PyAPI_FUNC`, :c:macro:`PyAPI_DATA` and
```

Reply:
Since they are not defined yet, Sphinx does not know that these macros have parameters.